### DISCUSSION TITLE
Mutual TLS  client certificates for webdav shares

### DISCUSSION THREAD

**Discussion by ippocratis** [floccusaddon/floccus#1412] [Ideas]
My webdav server requires a client cert

An ui option to provide a client certificate for webdav shares that are behind Mutual TLS would great addition.

Thanks

**Comment by JitteryDoodle** [comment]
This would be fantastic as I've locked down everything with mTLS and it broke floccus. Is this going to be planned? Otherwise I need to base it on the user agent of floccus which would defeat the purpose of mTLS.
